workspaces/orchestrator/plugins/orchestrator/src/components/WorkflowInstancePageContent.tsx

```diff
@@ -140,6 +142,28 @@ export const WorkflowInstancePageContent: React.FC<{
     setIsVariablesDialogOpen(prev => !prev);
   }, []);
 
+  const generic = usePermission({
```

Comment:
```suggestion
  const adminView = usePermission({
```
or similar self-explanatory name